src/playGame.js

```diff
@@ -0,0 +1,25 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+import { GAME_MESSAGES, RULES } from "../constants/messages.js";
+import randomNumber from "../utils/randomNumber.js";
+import userInputValidation from "../utils/userInputValidation.js";
+import resultPrint from "../utils/resultPrint.js";
+import compareToNumber from "../utils/compareToNumber.js";
+
+const playGame = async () => {
+  const comNum = randomNumber();
```

Comment:
```suggestion
  const computerNumbers = randomNumber();
```
이건 개인적인 취향일 수 있지만 저는 축약하지 않아도 괜찮을 것 같아요!  정말 축약한다면 `computerNum` 정도?! 

지금 `const comNum = randomNumber();` 되어있는데 저희는 요구사항을 알기 때문에 저게 컴퓨터의 숫자라는 것을 유추할 수 있을 것 같은데 모르는 사람이 보면 `comNum`에 대해 헷갈릴 수 있을 것 같아요🤔